runtime/rasdump/javadump.cpp

```diff
@@ -4338,7 +4342,16 @@ JavaCoreDumpWriter::writeMonitorObject(J9ThreadMonitor* monitor, j9object_t obj,
 	/* Describe its owning thread */
 	bool inflated = J9_ARE_ANY_BITS_SET(lock->flags, J9THREAD_MONITOR_INFLATED);
 
-	if ((NULL != owner) || (NULL != lockOwner)) {
+	if (IS_J9_OBJECT_MONITOR_OWNER_DETACHED(lockOwner)) {
+		if (inflated) {
+			_OutputStream.writeCharacters("owner \"");
+		} else {
+			_OutputStream.writeCharacters("Flat locked by \"");
+		}
+		_OutputStream.writeCharacters("<detached virtual thread>");
+		_OutputStream.writeCharacters(", entry count ");
+		_OutputStream.writeInteger(count, "%zu");
+	} else if ((NULL != owner) || (NULL != lockOwner)) {
```

Comment:
This is same as else if case except the `"<detached virtual thread>"`
I would suggest:
```
if (NULL != owner) &&... {
    if (inflated) {
        ...
    }

    if (IS_J9_OBJECT_MONITOR_OWNER_DETACHED(lockOwner)) {
        _OutputStream.writeCharacters("<detached virtual thread>");
    } else {
        line 4360 - 4378
    }

    _OutputStream.writeCharacters("), entry count ");
    _OutputStream.writeInteger(count, "%zu");
}
```

Reply:
The current change don't have the following for detached virtual thread. There is no J9VMThread or native thread.

```
if (NULL != owner) {
	_OutputStream.writeCharacters("J9VMThread:");
	_OutputStream.writePointer((void*)owner);
} else {
	_OutputStream.writeCharacters("native thread ID:");
	_OutputStream.writeInteger(omrthread_get_osId(lockOwner));
}
```

Reply:
This isn't what I was referring to, but Im good with this change.